pkg-precompile.jl

```diff
@@ -277,6 +291,19 @@ cache_paths = filter!(within_depot, compilecache_paths(env))
     "Precompile files to transfer (new additions $num_new/$total):\n$(join(paths, '\n'))"
 end
 
+# # Listing all cached precompile files can be useful in debugging unexpected failures but
+# # it can be extremely verbose.
+# @debug let paths = String[]
+#     for (root, dirs, files) in walkdir(joinpath(DEPOT_PATH[1], "compiled", "v$(VERSION.major).$(VERSION.minor)"))
+#         for file in files
+#             if endswith(file, ".ji")
+#                 push!(paths, joinpath(root, file))
+#             end
+#         end
+#     end
+#     "All precompile files within the cache mount:\n$(join(paths, '\n'))"
+# end
```

Comment:
This was useful in determining the source of the problem but is far too verbose for typical debug logs